proto/rpc.proto

```diff
@@ -9,6 +9,9 @@ message Invite {
   optional string roleName = 4;
   optional string roleDescription = 5;
   string invitorName = 6;
+  // TODO: Should this be required?
+  optional string projectColor = 7;
```

Comment:
highlighting this as related to my question about making this a required field

Reply:
Makes sense! Kind of hesitant to commit to using a default value in core though - is that because of the app versioning mismatch issue?

Reply:
based on discussion, this has to be optional for compat reasons. todo comment is no longer necessary